internal/controller/spectrumxrailpoolconfig_host_flows_controller.go

```
@@ -413,50 +413,27 @@ func updateDaemonsetNodeSelector(obj *uns.Unstructured, nodeSelector map[string]
-			Uplinks: []sriovv1.OVSUplinkConfigExt{uplink},
-		})
-	}
+	pfNames := lib.FilterNICs(ctx, rt.NicSelector.PfNames)
 
-	currentBridges, err := r.bridge.DiscoverBridges()
-	if err != nil {
-		return fmt.Errorf("failed to discover bridges: %w", err)
-	}
+	// Build desired br-railX bridge configs — one bridge per NIC
+	// Use ConfigureBridges from sriov-network-operator to manage them via OVSDB.
+	for idx, pfName := range pfNames {
```

golangci-lint (2.11.4)

[error] unnecessary leading newline

(whitespace)